content/hi/agile_software_development.md

```diff
@@ -0,0 +1,14 @@
+---
+title: एजाइल सॉफ्टवेयर विकास
+status: Completed
+category: अवधारणा
+---
+
+## यह क्या है
+प्रथाओं का एक सेट जो पुनरावृत्त विकास चक्रों और स्वयं-संगठित टीमों पर जोर देता है। जलप्रपात जैसी परियोजनाओं के विपरीत जहां मूल्य केवल एक परियोजना के अंत में उत्पन्न होता है, एजाइल सॉफ्टवेयर विकास मूल्य की निरंतर, वृद्धिशील वितरण और प्रक्रिया के विकासवादी सुधार पर केंद्रित है।
```

Comment:
I guess if we are using the word `एजाइल` we can also use waterfall in the same way.

Reply:
My view "Agile software development" is a commonly used phrase in Industry. Also, the translation for Agile doesn't makes the same sense.
Whereas in case of "waterfall like", its just commonly used English phrase and "जलप्रपात जैसी" means the same.
Thoughts?

Reply:
In this case, जलप्रपात जैसी makes sense. We could use वॉटर्फॉल if it were waterfall model in original text.